scripts/crawler.js

```diff
@@ -18,8 +18,25 @@ const V8_NATIVE_CALL_REPLACEMENT_PREFIX = '__v8_native_remainder'
 const V8_NATIVE_CALL_REPLACEMENT_REGEX = /__v8_native_remainder(\w+\(\S*?|\s*\))/gm
 const V8_NATIVE_CALL_FLAGS_COMMENT_REGEX = /\/\/\s*Flags:.*(--allow-natives-syntax)+/gm
 
-const GLOBAL_METHODS =
-  ";(function(globals){globals._ddiast = globals._ddiast || {plusOperator(res) {return res;}};}((1,eval)('this')));"
+const CSI_METHODS = [
+  { src: 'plusOperator' },
```

Comment:
What do you think if we try to do something different for `+` operation? Maybe `'#plusOperator'` or other special key instead of `src` something like `{ plusOperator: true }`
It looks like `plusOperator` is other method more, and not an operator.